Xamarin.Forms.Core/Button.cs

```diff
@@ -143,33 +136,33 @@ public Color TextColor
 			set { SetValue(TextElement.TextColorProperty, value); }
 		}
 
-		bool IButtonController.IsEnabledCore
+		bool IButtonElement.IsEnabledCore
 		{
 			set { SetValueCore(IsEnabledProperty, value); }
 		}
 
 		[EditorBrowsable(EditorBrowsableState.Never)]
-		public void SendClicked() => ButtonElementManager.ElementClicked(this, this);
+		public void SendClicked() => ButtonElement.ElementClicked(this, this);
 
 		public bool IsPressed => (bool)GetValue(IsPressedProperty);
 
 		[EditorBrowsable(EditorBrowsableState.Never)]
-		void IButtonController.SetIsPressed(bool isPressed) => SetValue(IsPressedPropertyKey, isPressed);
+		void IButtonElement.SetIsPressed(bool isPressed) => SetValue(IsPressedPropertyKey, isPressed);
```

Comment:
`SetIsPressed` is set from the platforms. its place is in the `Controller` interface

Reply:
*SetIsPressed* is only called from the Core code
It's called from 

```
		public static void ElementPressed(VisualElement visualElement, IButtonElement ButtonElementManager)
		{
			if (visualElement.IsEnabled == true)
			{
				ButtonElementManager.SetIsPressed(true);
				visualElement.ChangeVisualStateInternal();
				ButtonElementManager.PropagateUpPressed();
			}
		}

		public static void ElementReleased(VisualElement visualElement, IButtonElement ButtonElementManager)
		{
			if (visualElement.IsEnabled == true)
			{
				IButtonController buttonController = ButtonElementManager as IButtonController;
				ButtonElementManager.SetIsPressed(false);
				visualElement.ChangeVisualStateInternal();
				ButtonElementManager.PropagateUpReleased();
			}
		}
```